index.md

```diff
@@ -1,5 +1,4 @@
 ---
-title: Welcome to Pulumi
```

Comment:
This way, the home page's title will be "Pulumi" instead of "Welcome to Pulumi - Pulumi".

Reply:
We should probably match the www site for title here. This is very important to nail for SEO.

Reply:
Yes, this does match the www title here. It's just "Pulumi".